docs/content/features/auto-responses/index.md

```diff
@@ -180,6 +188,108 @@ In the event that architus is unable to parse your trigger regex, it will give a
+
+Example:
+```py
+book = struct(author="Frank Herbert", title="Dune", pages=818, reviews=["great", "amazing"])
+assert(book.author=="Frank Herbert")
+assert(book.pages==818)
+```
+
+#### Sum
+The `sum` function will sum all of the elements of an iterable. It accepts a single parameter that is the iterable value to sum over. `sum` requires that the parameter passed in is solely composed of numbers: integers, floats, or both. If only integers are in the iterable, it will return an integer. However, if the iterable is composed of floats or integers and floats, it will return a float value. The sum will always start at 0 and sum from there.
+
+Example:
+```py
+nums = range(1,101)
+values = [1,1.5,3,4.5]
+assert(sum(nums) == 5050)
+assert(sum(values) == 10)
+```
+
+### Global Variables
+Architus adds several global variables to the script environment to allow users to access information about the message, author, and channel that triggered the auto-response.
+
+`message` is a struct with members:
+- id: an integer that represents the discord id of the triggering message
+- content: a string that represents the raw content of the triggering message
+- clean: a string that is the clean content of the triggering message (this is much closer to how users see a message)
+
+`author` is a struct with members:
+- id: an integer that is the discord id of the author that wrote the triggering message
+- avatar\_url: a string that is a url for the author's avatar image
+- color: a string that is the color that the author's name is displayed using
+- discrim: an integer that is the discriminator part of the author's username
+- roles: a list of strings that contains the integer id values of each role the author is a part of
+- name: a string value of the author's username (does not include the discriminator)
+- nick: a string that is the author's nickname in the server in which the triggering message was sent
+- disp: a string that is the author's display name in the server in which the triggering message was sent
+
+`channel` is a struct with members:
+- id: an integer that is the discord id of the channel in which the triggering message was sent
+- name: a string that is the name of the channel in which the triggering message was sent
+
+`count` is a global variable that is just an integer of how many times the auto response has been triggered.
+
+`caps` is an array of strings that contain all of the capture groups defined in the regular expression trigger of the auto response. The order of the capture groups will be in the same order in which they were defined in the regex.
+
 ## Settings
 
 The auto response settings pane can be accessed by the `settings responses` command
```

Comment:
also add:
Allow Embeds, bool, If false, architus will escape links in responses so discord won't embed them
Allow Newlines, bool, Whether newlines will be stripped or allowed in auto response outputs